change/@fluentui-web-components-ee0dceff-c555-429c-b688-92aa40dc7f5e.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Accordion and Accordion Item init",
```

This message will appear in the changelog, please follow https://github.com/microsoft/fluentui/blob/ac0a425a4c2f900a051028a3594bdef0d61615af/rfcs/convergence/changelog-guidelines.md
```suggestion
  "comment": "feat(accordion): Add accordion web component",
```